fs/dafsserver/dafsserver.cpp

```diff
@@ -4919,6 +4932,10 @@ class CRemoteFileServer : implements IRemoteFileServer, public CInterface
         Owned<IRemoteActivity> outputActivity;
         OpenFileInfo fileInfo;
 
+        ISpan* requestSpan = queryNullSpan();
+        if (fileInfo.remoteRequest)
+            requestSpan = fileInfo.remoteRequest->queryRequestSpan();
```

Comment:
fileInfo.remoteRequest is always going to be null here..
Should this be moved ?